i18n/es.json

```diff
@@ -3398,10 +3402,22 @@
     "id": "ent.ldap.validate_filter.app_error",
     "translation": "Filtro de AD/LDAP inválido"
   },
+  {
+    "id": "ent.message_export.global_relay.attach_file.app_error",
+    "translation": "No se pudo añadir el adjunto al fichero de exportación de Global Relay."
+  },
+  {
+    "id": "ent.message_export.global_relay.close_zip_file.app_error",
+    "translation": "No se cerró correctamente el archivo zip."
+  },
   {
     "id": "ent.message_export.global_relay.create_file_in_zip.app_error",
     "translation": "No se puede crear el archivo eml."
   },
+  {
+    "id": "ent.message_export.global_relay.generate_email.app_error",
+    "translation": "No se pudo generar la data del archivo eml."
```

Comment:
I replaced the places where "the data" is translated as "la data" because for me (In Spain) is more natural to talk about "los datos" instead of "la data".